site/docs/v1/tech/oauth/url-validation.adoc

```diff
@@ -0,0 +1,274 @@
+---
+layout: doc
+title: URL Validation
+description: A description OAuth URL validation policies in FusionAuth
+navcategory: login-methods
+---
+
+[NOTE.since]
+====
+Available since 1.43.0
+====
+
+== Overview
+
+Beginning in version `1.43.0` FusionAuth provides support for wildcards in OAuth 2.0 redirect URLs and origin URLs. This document provides details on where wildcards are allowed in configured values and the valid replacement patterns for wildcards in each position.
+
+* <<URL Validation Policy>>
+* <<Allowed Wildcard Positions>>
+** <<Domain>>
+** <<Port>>
+** <<Path Segments>>
+** <<Query String Values>>
+* <<Wildcard Replacement Patterns>>
+** <<Domain>>
+** <<Port>>
+** <<Path Segments>>
+** <<Query String Values>>
+
+== URL Validation Policy
+
+In order to configured authorized redirect and origin URLs with wildcards, the [field]#URL validation# setting must be configured to `Allow wildcards` under [breadcrumb]#Applications -> Edit Application -> OAuth#.
+
+See the link:/docs/v1/tech/apis/applications[Application API] or [Application OAuth Configuration](/docs/v1/tech/core-concepts/applications#oauth) for details.
```

Comment:
This second link is broken.

Reply:
I didn't think `link:` worked when you wanted to link to a specific section on the page. That resolved it, but what was wrong with the syntax I had? I've seen it work in other places.

Reply:
I think he is talking about `[Application OAuth Configuration](/docs/v1/tech/core-concepts/applications#oauth)`

Reply:
Yeah. I changed it to a `link:` this morning. I thought the `[text](url)` pattern was another way to create links in AD/MD.